wbooks/src/app/screens/dashboard/screens/Detail/index.js

```diff
@@ -38,4 +39,12 @@ class BookDetail extends React.Component {
   }
 }
 
-export default BookDetail;
+BookDetail.propTypes = {
+  id: PropTypes.number
```

Comment:
`id` is never received directly as a param. You need to specify this on the `match` shape:
```
match: PropTypes.shape({
  params: PropTypes.shape({
    id: PropTypes.string.isRequired
  }).isRequired
})
```